plop-templates/basic-component/package.json.hbs

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@kaizen/{{kebabCase componentName}}",
+  "version": "1.0.0",
+  "description": "The {{titleCase componentName}} component",
```

Comment:
One issue we've had in the past is that there's been a tendency to create a new package for every new component, without thinking about whether it belongs in an existing package, or whether it makes sense to create a package that will allow for grouping.

I'm not sure if this is feasible at all, but would there be a way to give users a prompt to check if they want to add the component to an existing package, or create a new package when they use this?

Reply:
We can definitely add more prompts and further template configs to cater for that :) I just figured that would be the next iteration. But I could experiment on that next learning day to make sure it works.